tests/integration/forgejo/test_service.py

```diff
@@ -0,0 +1,43 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import pyforgejo
+import pytest
+from requre.online_replacing import (
+    record,
+)
+
+
+@pytest.mark.parametrize(
+    "kwargs_",
+    [
+        pytest.param(
+            {"repo": "test", "namespace": "packit"},
+            id="create project with namespace",
+        ),
+        pytest.param(
+            {"repo": "test", "namespace": None},
+            id="create project without namespace",
+        ),
+        pytest.param(
+            {"repo": "test_1", "namespace": None, "description": "A repo description"},
+            id="create project with description",
+        ),
+    ],
+)
+@record(what="pyforgejo.core.http_client.HttpClient.request")
+def test_project_create(service, kwargs_):
```

Comment:
I used `pytest+params` here instead of `unittest+inheritance` because I like the way pytest can save us code duplication. But I am fine to go back using unittest for maintaining code compatibility in tests with other modules.